README.md

```diff
@@ -157,3 +157,26 @@ Common usage:
   ```shell
   docker run --rm -v ~/solid-config:/config -p 3000:3000 -it css:latest -c /config/my-config.json
   ```
+
+## Using the identity provider
+
+1. First launch the Community Solid Server (see [Running locally](#running-locally)).
+2. To use the identity provider, you need a compatible client application.
+
+    You can use for example `@inrupt/solid-client-authn-js`:
+
+    ```bash
+    git clone https://github.com/inrupt/solid-client-authn-js
+    cd solid-client-authn-js
+    npm ci
+    cd packages/node/example/demoClientApp/
```

Comment:
I also had to run `npm ci` after this command as the demo client app wasn't built by the root `npm ci` call. Is this expected?

Reply:
You're right, I haven't done it in a while since there was no change to the demo client app recently.
Fixed in the last commit.